electron-devtools-installer/electron-devtools-installer-tests.ts

```diff
@@ -0,0 +1,35 @@
+/// <reference path="../github-electron/github-electron.d.ts" />
+/// <reference path="electron-devtools-installer.d.ts" />
+
+import installExtension, {
+    EMBER_INSPECTOR, REACT_DEVELOPER_TOOLS,
+    BACKBONE_DEBUGGER, JQUERY_DEBUGGER,
+    ANGULARJS_BATARANG, VUEJS_DEVTOOLS,
+    REDUX_DEVTOOLS, REACT_PERF,
+} from 'electron-devtools-installer';
+
+
+installExtension(EMBER_INSPECTOR)
```

Comment:
You can just do `const x: Promise<string> = installExtension(EMBER_INSPECTOR);`. Values interpolated in a template aren't typechecked as strings.